applications/Bright_Disputes.md

```diff
@@ -0,0 +1,180 @@
+
+
+We support startups, digital agencies as well as medium to big businesses. We cooperate with startups, accelerators and incubators. Whatever the client profile is, we always  aim to establish a satisfying partnership for both sides. Since 2012 we have built software for more than 40 businesses worldwide.
+
+### Team's experience
+As a company we have a 4 year experience with blockchain technologies, including cooperation with Parity. Since almost 2 years we’re developing our own Substrate based product [BrightTreasury](https://treasury.bright.dev/), that's been funded first by Web3 Foundation Grants program and has been supported by Polkadot and Kusama treasuries since. More recently, we’ve engaged in Rust development for Substrate as well, with two Web3 Foundation grants: [ZK-Snarks tutorial](https://github.com/w3f/Grants-Program/pull/1290) and [High-availability validator setup with Raft](https://github.com/w3f/Grant-Milestone-Delivery/pull/813).
+
+Who would work on Bright Disputes?
+- Michał - Rust Developer, for over 12 years he has been working as a C++ developer. In his career he participate in various projects, starting from the embedded devices, automotive, mobile games and ending on the financial systems. Right now he is focused on the blockchain technology.
+- Kasia - she has been working with agile methods for over 10 years, both as a researcher and a practitioner. She believes in teamwork and a power of user centered mindset. With experience in leading international projects, she knows how to support and encourage timely and high quality deliveries.
+
+### Team Code Repos
+
+-  https://github.com/bright
+
+Team Members:
+-  Michał: https://github.com/mgralinski-bright
+-  Kasia: https://github.com/bufta
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/michal-gralinski-b464152/
+- https://www.linkedin.com/in/katarzyna-łukasiewicz-b473901aa/
+
+
+## Development Status :open_book:
+
+The result of our research regarding the solutions and architecture has been described in this proposal.
+
+## Development Roadmap :nut_and_bolt:
+
+
+
+### Overview
+
+- **Total Estimated Duration:** 4,5 months
+- **Full-Time Equivalent (FTE):** 1,1
+- **Total Costs:** 42 000 USD
+
+### Milestone 1 Example — Basic functionality
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,1
+- **Costs:**  10 000 USD
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation | We will provide **inline documentation** of the code. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Repository | We will create a GitHub repository for Dispute project.
```

Comment:
nitpick: This doesn't need to be an explicit deliverable

Reply:
No need to remove it